store/sqlstore/upgrade.go

```diff
@@ -126,6 +132,10 @@ func upgradeDatabase(sqlStore SqlStore, currentModelVersionString string) error
 		mlog.Warn("The database schema version and model versions do not match", mlog.String("schema_version", currentSchemaVersion.String()), mlog.String("model_version", currentModelVersion.String()))
 	}
 
+	// max execution time to execute upgrade statements
+	currentExecutionTimeout := getAndSetMaxExecutionTime(sqlStore, decideMaxExecutionTime(sqlStore))
```

Comment:
Should we fail in case we aren't able to raise the timeout? I think is more safe to just accept this and try to upgrade anyway, if there is a timeout is going to be anyway. Probably a log message would be great, but exiting the process is too much. What do you think @rvillablanca?

Reply:
Yep, I agree with you that exiting could be too much, so if I change it by adding an error log then  the intention of this PR would be an **attempt** to set the timeout. I like that approach so I will make the changes soon @jespino